openstates/in/utils.py

```diff
@@ -0,0 +1,22 @@
+import requests
+
+
+def get_with_increasing_timeout(scraper,link,fail=False,kwargs={}):
```

Comment:
OK, I'll check this out in the morning and see if I can get it working that way.